.changeset/some-rabbits-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sl-design-system/callout': patch
+---
+
+New `callout` component.
```

Comment:
Can you add a little bit more information on *why* we have callout and how it is different from inline message? The text here ends up in `CHANGELOG.md` :)